### ISSUE TITLE
Disable atomisp_ov5693 in config

### ISSUE THREAD

**Issue by fabwu** [linux-surface/linux-surface#287]
On my Surface Book 2 I get the following error because the staging driver for ov5693 on Atomisp systems is incorrectly probing the camera sensor on my device. 
```
[    2.027871] atomisp_ov5693: module is from the staging directory, the quality is unknown, you have been warned.
[    2.030710] ov5693 i2c-INT33BE:00: Failed to find EFI variable INT33BE:00_I2CAddr
[    2.030712] ov5693 i2c-INT33BE:00: I2CAddr: using default (-1)
...
[    2.034940] ov5693: probe of i2c-INT33BE:00 failed with error -22
```
What do you think about disable the driver via config?

**Comment by qzed** [comment]
Looks like this is enabled by the Arch and Debian v5.8 config of the default distribution kernel (which we use as base), Fedora doesn't seem to have that enabled. We can disable it for now via our extra-configs for those builds.

In the long term we'll probably have to find a better fix for this, specifically with regards to the camera driver development. We'd probably want something that prevents loading of the driver on non-atomisp platforms entirely, regardless if the driver is present in the kernel or not. The same goes for any ov5693 driver for the Surface cameras: That probably shouldn't try to load on an atomisp platform.

IMHO that's a question for the later phases of the Surface camera driver development though.

**Comment by qzed** [comment]
Fixed in 6298490.
